fpdf/graphics_state.py

```diff
@@ -158,3 +167,75 @@ def text_mode(self):
     @text_mode.setter
     def text_mode(self, v):
         self.__statestack[-1]["text_mode"] = TextMode.coerce(v)
+
+    @property
+    def char_vpos(self):
```

Comment:
Could you add docstrings on this/those property(ies) pointing to https://pyfpdf.github.io/fpdf2/TextStyling.html#subscript-superscript-and-fractional-numbers please?

Reply:
will do

Reply:
I'm using relative links, hope I got them right...